app/Member.php

```diff
@@ -3,7 +3,7 @@
 namespace App;
 
 use Illuminate\Database\Eloquent\Model;
-use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Database\Eloquent\SoftDeletes;	
```

Comment:
Was an extra space added here? Usually we try not to have trailing spaces (to the extent of installing a plugin for that), and it's also good to reduce noise in the diff (i.e. when looking at files changed, reviewers prefer to see only changes that are actually relevant to the diff).